challenges/CC_61_fractal_spirograph/CC_61_fractal_spirograph.pde

```diff
@@ -10,12 +11,13 @@ int resolution = 50;
 
 Orbit sun;
 Orbit end;
+Orbit next;
 
 void setup() {
   size(600, 600);
   path = new ArrayList<PVector>();
   sun = new Orbit(width/2, height/2, width/4, 0);
-  Orbit next = sun;
+  next = sun;
```

Comment:
Yes, of course, I just thought it made sense to make it a global variable instead of declaring it every time since we were using it in three different places, However, the `int i = 0` comparison is on point, I agree and have reverted the changes